CONTRIBUTING.md

```diff
@@ -0,0 +1,173 @@
+<!-- markdownlint-disable MD033 -->
+# Contributing to Paricia
+
+Thanks for taking the time to contribute to Paricia!
+
+The following is a set of guidelines for contributing to Paricia, a Python-based hydroclimatic data management system project. The goal of these guidelines is to make the development of the project efficient and sustainable and to ensure that every commit makes it better, more readable, more robust and better documented. Please, feel free suggest changes and improvements.
+
+(this guide is based on the [Atom editor guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md))
+
+## Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[How Can I Contribute?](#how-can-i-contribute)
+
+- [Reporting Bugs](#reporting-bugs)
+- [Suggesting Enhancements](#suggesting-enhancements)
+- [Your First Code Contribution](#your-first-code-contribution)
+- [Pull Requests](#pull-requests)
+
+[Styleguides](#styleguides)
+
+- [Git Commit Messages](#git-commit-messages)
+- [Documentation Styleguide](#documentation-styleguide)
+
+[Developer's setup](#developers-setup)
+
+- [Installing Paricia](#installing-paricia)
+- [Tests](#tests)
+- [Continuous integration](#continuous-integration)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [Paricia Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [the repository Administrator](mailto:).
```

Comment:
I've added a generic CoC in #255 , merging into this branch. About the contact, I've used Wouter's email there, and probably the same should go in here. @ICHydro , could you confirm you're happy to appear here?

Reply:
Sure. You can also link to my Imperial profile page, but email is file.